src/main/java/de/tum/in/www1/artemis/config/Constants.java

```diff
@@ -89,6 +89,12 @@ public final class Constants {
     // Used as a value for <sourceDirectory> for the Java template pom.xml
     public static final String STUDENT_WORKING_DIRECTORY = ASSIGNMENT_DIRECTORY + "src";
 
+    public static final String USER_FIRST_NAME_AFTER_SOFT_DELETE = "Anonymous";
+
+    public static final String USER_LAST_NAME_AFTER_SOFT_DELETE = "User";
+
+    public static final String USER_EMAIL_DOMAIN_AFTER_SOFT_DELETE = "@user.anonymous";
```

Comment:
I guess this is okay for debugging purposes, but the client should check the user if it is a deleted user and then insert a localized version of the name itself. I think Anonymous is a bad naming scheme, as I would associate an anonymized user to be a user that is not deleted but just does not want their name to be shown. I think "Deleted" would be a better name.